prettier/main_test.ts

```diff
@@ -233,3 +233,267 @@ test(async function testPrettierPrintToStdout(): Promise<void> {
 
   emptyDir(tempDir);
 });
+
+test(async function testPrettierReadFromStdinFormatTS(): Promise<void> {
+  const inputCode = `console.log("abc"  )`;
+  const p1 = Deno.run({
+    args: ["deno", "./prettier/testdata/echox.ts", `${inputCode}`],
+    stdout: "piped"
+  });
+
+  const p2 = Deno.run({
+    args: ["deno", "run", "./prettier/main.ts", "--stdin"],
+    stdin: "piped",
+    stdout: "piped"
+  });
+
+  if (!p2.stdin) {
```

Comment:
I think these checks of stdin and stdout are redundant because they are tested in deno core repository.

Reply:
> I think these checks of stdin and stdout are redundant because they are tested in deno core repository.

Make sense